src/content/docs/journey-app-slow/root-causes.mdx

```diff
@@ -0,0 +1,92 @@
+---
+title: Prepare to triage your application
+metaDescription: 
+---
+
+import journeyWebTrans from 'images/journey_screenshot-web-transactions.webp'
+
+Your app is slow. Now what? How do you track down the root cause of your app's slowness?
+
+This tutorial walks you through how to use New Relic to triage your application and identify the root cause of your app's degraded performance. You'll start by "instrumenting" your app with an agent, which means installing a piece of code that reports data about your app to New Relic. Using this data in New Relic, you'll dig into the performance of your application and identify slow transactions, slow database queries, or slow external services as your issue - or possibly all three!
+
+
+
+## Prerequisite: instrument your app [#prereq]
+
+Before you can improve your site's performance, you need the ability to see what's going on. New Relic monitoring always starts with an **integration**, or a piece of code that collects data from a relevant technology and reports it to New Relic. To monitor your application's performance, you will use an agent created specifically for your app's language.
+
+Click a logo to install an agent. This will send you to the New Relic platform where you will be guided through installing and configuring the agent.
+
+<TechTileGrid>
+  
+  <TechTile
+    name="Go agent"
+    icon="logo-go"
+    to="https://one.newrelic.com/nr1-core?state=985d4005-ba90-a8c7-1da1-2af34539b03b"
+  />
+
+  <TechTile
+    name="Java agent"
+    icon="logo-java"
+    to="https://one.newrelic.com/nr1-core?state=80d18bcb-4919-1fcb-2b77-9406838eb916"
+  />
+
+  <TechTile
+    name=".NET agent"
+    icon="logo-dotnet"
+    to="https://one.newrelic.com/nr1-core?state=30e93090-6dfa-6b70-8e75-472f54414355"
+  />
+
+  <TechTile
+    name="Node.js agent"
+    icon="logo-nodejs"
+    to="https://one.newrelic.com/marketplace/install-data-source?state=be2e62fa-cc3b-c428-27c4-8d662c9e80a1"
+  />
+
+  <TechTile
+    name="PHP agent"
+    icon="logo-php"
+    to="https://one.newrelic.com/nr1-core?state=aa633b41-72d4-009c-3abf-55dcf64894fe"
+  />
+
+  <TechTile
+    name="Python agent"
+    icon="logo-python"
+    to="https://one.newrelic.com/nr1-core?state=20fda75b-58fb-a92a-f9e1-7b052035c6e8"
+  />
+
+  <TechTile
+    name="Ruby agent"
+    icon="logo-ruby"
+    to="https://one.newrelic.com/nr1-core?state=d69143ab-605c-579b-25bf-cc6e5fee5b80"
+  />
+
+</TechTileGrid>
+
+Once you've installed an agent, go to **[one.newrelic.com](https://one.newrelic.com/nr1-core)** and select your app. If you don't see much data just yet, step away for a while and let the agent gather real-time data as your application runs.
+
+## Identify optimization opportunities [#identify]
+
+APM uses dozens of dashboards and charts. Let's talk about what you should focus on first:
```

Comment:
This is actually the first mention of APM in the doc. Perhaps a short mention above that you're installing a "language agent for APM" and explanation fo what that means?